src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -14,14 +16,15 @@ public class AuthenticationController {
     private final AuthenticationService authService;
     private final ResponseDtoMapper<UserResponseDto, User> userDtoResponseMapper;
 
+    @Autowired
```

Comment:
Please remove @Autowired through whole project in situations where we have only one constructor as dependency injection via constructor will work automatically in such case